src/settings.rs

```diff
@@ -57,6 +57,18 @@ Settings which only apply to certain [`Planner`](crate::planner::Planner)s shoul
 #[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
 #[cfg_attr(feature = "cli", derive(clap::Parser))]
 pub struct CommonSettings {
+    /// Attribute the diagnostics of this run to a specific usage category
+    #[cfg_attr(
+        feature = "cli",
+        clap(
+            long,
+            default_value = None,
+            env = "NIX_INSTALLER_ATTRIBUTION",
+            global = true
+        )
+    )]
+    pub attribution: Option<String>,
```

Comment:
I'd prefer it to be moved down to the bottom or second bottom alongside the like-value so that in `--help` displays related settings will be nearby.